internal/resource/nvml-device.go

```diff
@@ -99,3 +101,19 @@ func (d nvmlDevice) GetPCIClass() (uint32, error) {
 	}
 	return nvDevice.Class, nil
 }
+
+func (d nvmlDevice) GetFabricIDs() (string, string, error) {
```

Comment:
Why is no `IsFabricAttached()` method defined on an `nvmlDevice`? How does this even compile?

Reply:
is inherited https://github.com/NVIDIA/k8s-device-plugin/blob/main/internal/resource/nvml-device.go#L28

Reply:
it comes from PR https://github.com/NVIDIA/go-nvlib/pull/44

Reply:
Ack